examples/asr/transcribe_speech.py

```diff
@@ -93,6 +94,86 @@ class TranscriptionConfig:
     rnnt_decoding: RNNTDecodingConfig = RNNTDecodingConfig(fused_batch_size=-1)
 
 
+def transcribe_partial_audio(
```

Comment:
Where's the equivalent for RNNT ? If it's not supported, raise an error please. You can do this by asserting that obj is instance of only CTC classes.